.gitignore

```diff
@@ -1,6 +1,9 @@
 desktop.ini
 *.docx
 *.ico
+*.jpg
+*.png
```

Comment:
I'm curious, why do you include file types in the gitignore?

Reply:
Because I have some screenshots from the exercises. I store my courses in folders with materials, so I exclude that kind of material from the code. Of course, I´m not going to do that in a real project.